### PR TITLE
Use pykg2tbl inside Broker

### PR DESCRIPTION
**Pull request by JotaFan**
Replace SparqlWrapper for pykg2tbl.
Adding some testing on the changed functions.

dependent on:
https://github.com/envri-fair/marine-eov-broker/pull/20

### CODE REVIEW

File: src/marine_eov_broker/MarineRiBroker.py
Comment (marc-portier) [inline src/marine_eov_broker/MarineRiBroker.py]: `_data` ?? points to a guaranteed internal dict build from either rdflib or sqlwrapper source I presume?

however ties in with the dict based `QueryResult` implementation?
maybe cleaner to have/use some `to_dict()` on the `QueryResult`?

File: src/marine_eov_broker/MarineRiBroker.py
Comment (marc-portier) [inline src/marine_eov_broker/MarineRiBroker.py:461]: this is better --> see previous comment 
pls use `query_result.to_dict()` and not `query_result._data` to reduce dependency-surface

File: src/marine_eov_broker/MarineRiBroker.py
Comment (marc-portier) [inline src/marine_eov_broker/MarineRiBroker.py:440]: so after all no `submit_sparql_named_query` variant that also abuses the templating 
(further cleaning the ipynb code not worth it?)

File: src/marine_eov_broker/NVSQueries.py
Comment (marc-portier) [inline src/marine_eov_broker/NVSQueries.py:8]: remember we are still waiting on the feedback if the full list of generated sparql is acutally useful / correct

File: src/marine_eov_broker/j2_templates/eov_query.sparql
Comment (marc-portier) [inline src/marine_eov_broker/j2_templates/eov_query.sparql]: tabs versus spaces on lines 56-58 to fix indent

**Review by marc-portier — APPROVED** [review]
minor comments - non prohibiting a merge

File: src/marine_eov_broker/MarineRiBroker.py
Comment (JotaFan) [reply src/marine_eov_broker/MarineRiBroker.py:461]: Agree! (for both comments)

**Review by marc-portier — APPROVED** [review]

File: src/marine_eov_broker/MarineRiBroker.py
Comment (JotaFan) [reply src/marine_eov_broker/MarineRiBroker.py:440]: Added now.
Not sure if ipynb cleaning is needed though.
The best way to implement `submit_sparql_named_query` would also imply a refactor in most of the function the `submit_query` uses

Comment (marc-portier) [reply src/marine_eov_broker/MarineRiBroker.py:440]: yeah, I rest my case, goes outside the scope of what we aim for now

**Comment by JotaFan** [comment]
@galviset The packages are all up and running, feel free to merge this one after review

**Pull request merged**
